packages/url_launcher/url_launcher/lib/src/url_launcher_uri.dart

```diff
@@ -37,6 +37,16 @@ import 'type_conversion.dart';
 ///  - "_self" opens the new URL in the current tab.
 /// Default behaviour when unset is to open the url in a new tab.
 ///
+/// Some web browsers, such as Safari, may prevent URL launching if it is not
```

Comment:
I normally add these notes to the README of the web package under a "Limitations of the web platform" section ([example](https://source.corp.google.com/piper///depot/google3/third_party/dart/devtools_app/lib/src/shared/config_specific/framework_initialize/_framework_initialize_web.dart;l=6), very similar to the **Autoplay** problem!), but I'm not sure if those are discoverable enough, since they're by default "hidden" in pub

Reply:
Yes, I agree that it's a bit hidden and the `README.md` would also a good place 👍 If you or @stuartmorgan like the approach with `README.md` more, I could copy the structure of the `video_player` package (having a short note that there are limitations on web in the main README and a more detailed description in `url_launcher_web` package).

Reply:
I think putting the long explanation in the README and then referencing that here is probably the best option.

Reply:
I made the following changes:

* Adding a link about the web limitations in the "Configuration" section of the `README.md` of `url_launcher` (like `video_player` has), please let me know if this is the wrong location
* Adding a new section "Limitations on the Web platform" with the detailed documentation in the `README.md` of the `url_launcher_web` package
* Just referencing to this limitation in the `launchUrl` method